src/constants/actionTypes.js

```diff
@@ -1,10 +1,8 @@
 "use strict";
 
-var keyMirror = require('react/lib/keyMirror');
-
-module.exports = keyMirror({
-	INITIALIZE: null,
-	CREATE_AUTHOR: null,
-	UPDATE_AUTHOR: null,
-	DELETE_AUTHOR: null
-});
\ No newline at end of file
+module.exports = {
+	INITIALIZE: 'INITIALIZE',
+	CREATE_AUTHOR: 'CREATE_AUTHOR',
+	UPDATE_AUTHOR: 'UPDATE_AUTHOR',
+	DELETE_AUTHOR: 'DELETE_AUTHOR'
+};
```

Comment:
keymirror is no longer shipped with React Router. Eliminated use since unnecessary.